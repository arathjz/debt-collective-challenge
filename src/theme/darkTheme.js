module.exports = {
  main: '#162447',
  secondary: '#1f4068',
  alternary: '#1b1b2f',
  textColor: '#f0f0f0',
};
