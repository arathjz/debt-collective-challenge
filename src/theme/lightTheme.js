module.exports = {
  main: '#f0f0f0',
  secondary: '#1b6ca8',
  alternary: '#ffffff',
  textColor: '#162447',
};
