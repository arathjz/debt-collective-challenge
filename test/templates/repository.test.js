/* eslint-disable react/jsx-pascal-case */
import React from 'react';
import { useStaticQuery } from 'gatsby';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { ThemeProvider } from 'styled-components';

import Repository from '../../src/templates/repository';
import theme from '../../src/theme/darkTheme';
import repositoryFixture from '../fixtures/repository.json';

jest.mock('../../src/hooks/useOrganization', () => {
  return () => ({
    github: {
      organization: {
        name: 'The debt collective',
        avatarUrl: 'https://images.com/image.svg',
        login: 'debtcollective',
        url: 'https://github.com/debtcollective',
      },
    },
  });
});

describe('Repository template', () => {
  const { repository } = repositoryFixture.github;

  beforeEach(() => {
    useStaticQuery.mockImplementation(() => ({
      placeholderImage: {
        childImageSharp: {
          fixed: {
            base64: 'image',
            width: 50,
            height: 50,
            src: '/static/image.png',
            srcSet: '/static/image.png',
          },
        },
      },
    }));
  });

  test('render correctly', () => {
    const container = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });

  test('display title', () => {
    const { getByText } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByText(repository.name)).toBeInTheDocument();
  });

  test('display repository url', () => {
    const { getByText } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByText(repository.url)).toBeInTheDocument();
  });

  test('display repository description', () => {
    const { getByText } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByText(repository.description)).toBeInTheDocument();
  });

  test('display total stars', () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByTestId('stars')).toBeInTheDocument();
    expect(getByTestId('stars')).toHaveTextContent(repository.stargazers.totalCount);
  });

  test('display total forks', () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByTestId('forks')).toBeInTheDocument();
    expect(getByTestId('forks')).toHaveTextContent(repository.forks.totalCount);
  });

  test('display total pullRequests', () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByTestId('pullRequests')).toBeInTheDocument();
    expect(getByTestId('pullRequests')).toHaveTextContent(repository.pullRequests.totalCount);
  });

  test('display total watchers', () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByTestId('watchers')).toBeInTheDocument();
    expect(getByTestId('watchers')).toHaveTextContent(repository.watchers.totalCount);
  });

  test('display total issues', () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <Repository data={repositoryFixture} />
      </ThemeProvider>,
    );

    expect(getByTestId('issues')).toBeInTheDocument();
    expect(getByTestId('issues')).toHaveTextContent(repository.issues.totalCount);
  });
});
