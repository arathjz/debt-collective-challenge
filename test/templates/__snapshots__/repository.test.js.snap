// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repository template render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-AxhCb BIYNM"
      >
        <header
          class="sc-AxiKw bQuuop"
        >
          <div
            class="content"
          >
            <a
              class="sc-AxirZ cLMVqT"
              href="/"
            >
              <div
                class=" gatsby-image-wrapper"
                style="position: relative; overflow: hidden; display: inline-block; width: 50px; height: 50px;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="image"
                  style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
                />
                <picture>
                  <source
                    srcset="/static/image.png"
                  />
                  <img
                    alt=""
                    height="50"
                    loading="lazy"
                    src="/static/image.png"
                    srcset="/static/image.png"
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
                    width="50"
                  />
                </picture>
                <noscript>
                  &lt;picture&gt;&lt;source srcset="/static/image.png" /&gt;&lt;img loading="lazy" width="50" height="50" srcset="/static/image.png" src="/static/image.png" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                </noscript>
              </div>
              <h1 />
            </a>
            <button
              aria-label="Left Align"
              class="sc-AxjAm esyKCT"
              type="button"
            >
              <img
                alt="toggle dark/light theme"
                src="test-file-stub"
              />
            </button>
          </div>
        </header>
        <div>
          <a
            class="sc-AxheI oCzJw"
            href="/repositories"
          >
            ‚Üê Go back to repositories list
          </a>
          <section
            class="sc-AxgMl bQtgzz"
          >
            <h3>
              <span>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="git"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-git-alt fa-w-14"
                    data-icon="git-alt"
                    data-prefix="fab"
                    focusable="false"
                    height="36"
                    role="img"
                    viewBox="0 0 448 512"
                    width="36"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                 
              </span>
              fundraising
               
               
            </h3>
            <a
              href="https://github.com/debtcollective/fundraising"
              rel="noreferrer"
              target="_blank"
            >
              <span
                class="sc-AxhUy hejoPV"
                data-testid="link"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-link fa-w-16"
                  data-icon="link"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 512 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span>
                https://github.com/debtcollective/fundraising
              </span>
            </a>
            <div
              class="repository-data"
            >
              <p>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="star"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-star fa-w-18"
                    data-icon="star"
                    data-prefix="fas"
                    focusable="false"
                    height="14"
                    role="img"
                    viewBox="0 0 576 512"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  data-testid="stars"
                >
                  0
                </span>
              </p>
              <p>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="fork"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-utensils fa-w-13"
                    data-icon="utensils"
                    data-prefix="fas"
                    focusable="false"
                    height="14"
                    role="img"
                    viewBox="0 0 416 512"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M207.9 15.2c.8 4.7 16.1 94.5 16.1 128.8 0 52.3-27.8 89.6-68.9 104.6L168 486.7c.7 13.7-10.2 25.3-24 25.3H80c-13.7 0-24.7-11.5-24-25.3l12.9-238.1C27.7 233.6 0 196.2 0 144 0 109.6 15.3 19.9 16.1 15.2 19.3-5.1 61.4-5.4 64 16.3v141.2c1.3 3.4 15.1 3.2 16 0 1.4-25.3 7.9-139.2 8-141.8 3.3-20.8 44.7-20.8 47.9 0 .2 2.7 6.6 116.5 8 141.8.9 3.2 14.8 3.4 16 0V16.3c2.6-21.6 44.8-21.4 48-1.1zm119.2 285.7l-15 185.1c-1.2 14 9.9 26 23.9 26h56c13.3 0 24-10.7 24-24V24c0-13.2-10.7-24-24-24-82.5 0-221.4 178.5-64.9 300.9z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  data-testid="forks"
                >
                  0
                </span>
              </p>
              <p>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="pullRequest"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-code-branch fa-w-12"
                    data-icon="code-branch"
                    data-prefix="fas"
                    focusable="false"
                    height="14"
                    role="img"
                    viewBox="0 0 384 512"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M384 144c0-44.2-35.8-80-80-80s-80 35.8-80 80c0 36.4 24.3 67.1 57.5 76.8-.6 16.1-4.2 28.5-11 36.9-15.4 19.2-49.3 22.4-85.2 25.7-28.2 2.6-57.4 5.4-81.3 16.9v-144c32.5-10.2 56-40.5 56-76.3 0-44.2-35.8-80-80-80S0 35.8 0 80c0 35.8 23.5 66.1 56 76.3v199.3C23.5 365.9 0 396.2 0 432c0 44.2 35.8 80 80 80s80-35.8 80-80c0-34-21.2-63.1-51.2-74.6 3.1-5.2 7.8-9.8 14.9-13.4 16.2-8.2 40.4-10.4 66.1-12.8 42.2-3.9 90-8.4 118.2-43.4 14-17.4 21.1-39.8 21.6-67.9 31.6-10.8 54.4-40.7 54.4-75.9zM80 64c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16zm0 384c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zm224-320c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  data-testid="pullRequests"
                >
                  180
                </span>
              </p>
              <p>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="watch"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-eye fa-w-18"
                    data-icon="eye"
                    data-prefix="fas"
                    focusable="false"
                    height="14"
                    role="img"
                    viewBox="0 0 576 512"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  data-testid="watchers"
                >
                  2
                </span>
              </p>
              <p>
                <span
                  class="sc-AxhUy hejoPV"
                  data-testid="issue"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-info-circle fa-w-16"
                    data-icon="info-circle"
                    data-prefix="fas"
                    focusable="false"
                    height="14"
                    role="img"
                    viewBox="0 0 512 512"
                    width="14"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  data-testid="issues"
                >
                  74
                </span>
              </p>
            </div>
            <p>
              Fundraising App
            </p>
          </section>
        </div>
        <footer>
          <p>
            Icons powered by
             
            <a
              href="https://fontawesome.com/license"
              rel="noreferrer"
              target="_blank"
            >
              FontAwesome
            </a>
          </p>
          <p>
            Build with ‚ô• by
             
            <a
              href="https://github.com/arathjz"
              rel="noreferrer"
              target="_blank"
            >
              @arathjz
            </a>
             
          </p>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-AxhCb BIYNM"
    >
      <header
        class="sc-AxiKw bQuuop"
      >
        <div
          class="content"
        >
          <a
            class="sc-AxirZ cLMVqT"
            href="/"
          >
            <div
              class=" gatsby-image-wrapper"
              style="position: relative; overflow: hidden; display: inline-block; width: 50px; height: 50px;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="image"
                style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
              />
              <picture>
                <source
                  srcset="/static/image.png"
                />
                <img
                  alt=""
                  height="50"
                  loading="lazy"
                  src="/static/image.png"
                  srcset="/static/image.png"
                  style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
                  width="50"
                />
              </picture>
              <noscript>
                &lt;picture&gt;&lt;source srcset="/static/image.png" /&gt;&lt;img loading="lazy" width="50" height="50" srcset="/static/image.png" src="/static/image.png" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
              </noscript>
            </div>
            <h1 />
          </a>
          <button
            aria-label="Left Align"
            class="sc-AxjAm esyKCT"
            type="button"
          >
            <img
              alt="toggle dark/light theme"
              src="test-file-stub"
            />
          </button>
        </div>
      </header>
      <div>
        <a
          class="sc-AxheI oCzJw"
          href="/repositories"
        >
          ‚Üê Go back to repositories list
        </a>
        <section
          class="sc-AxgMl bQtgzz"
        >
          <h3>
            <span>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="git"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-git-alt fa-w-14"
                  data-icon="git-alt"
                  data-prefix="fab"
                  focusable="false"
                  height="36"
                  role="img"
                  viewBox="0 0 448 512"
                  width="36"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M439.55 236.05L244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"
                    fill="currentColor"
                  />
                </svg>
              </span>
               
            </span>
            fundraising
             
             
          </h3>
          <a
            href="https://github.com/debtcollective/fundraising"
            rel="noreferrer"
            target="_blank"
          >
            <span
              class="sc-AxhUy hejoPV"
              data-testid="link"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-link fa-w-16"
                data-icon="link"
                data-prefix="fas"
                focusable="false"
                height="14"
                role="img"
                viewBox="0 0 512 512"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
                  fill="currentColor"
                />
              </svg>
            </span>
            <span>
              https://github.com/debtcollective/fundraising
            </span>
          </a>
          <div
            class="repository-data"
          >
            <p>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="star"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star fa-w-18"
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 576 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                data-testid="stars"
              >
                0
              </span>
            </p>
            <p>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="fork"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-utensils fa-w-13"
                  data-icon="utensils"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 416 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M207.9 15.2c.8 4.7 16.1 94.5 16.1 128.8 0 52.3-27.8 89.6-68.9 104.6L168 486.7c.7 13.7-10.2 25.3-24 25.3H80c-13.7 0-24.7-11.5-24-25.3l12.9-238.1C27.7 233.6 0 196.2 0 144 0 109.6 15.3 19.9 16.1 15.2 19.3-5.1 61.4-5.4 64 16.3v141.2c1.3 3.4 15.1 3.2 16 0 1.4-25.3 7.9-139.2 8-141.8 3.3-20.8 44.7-20.8 47.9 0 .2 2.7 6.6 116.5 8 141.8.9 3.2 14.8 3.4 16 0V16.3c2.6-21.6 44.8-21.4 48-1.1zm119.2 285.7l-15 185.1c-1.2 14 9.9 26 23.9 26h56c13.3 0 24-10.7 24-24V24c0-13.2-10.7-24-24-24-82.5 0-221.4 178.5-64.9 300.9z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                data-testid="forks"
              >
                0
              </span>
            </p>
            <p>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="pullRequest"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-code-branch fa-w-12"
                  data-icon="code-branch"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 384 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M384 144c0-44.2-35.8-80-80-80s-80 35.8-80 80c0 36.4 24.3 67.1 57.5 76.8-.6 16.1-4.2 28.5-11 36.9-15.4 19.2-49.3 22.4-85.2 25.7-28.2 2.6-57.4 5.4-81.3 16.9v-144c32.5-10.2 56-40.5 56-76.3 0-44.2-35.8-80-80-80S0 35.8 0 80c0 35.8 23.5 66.1 56 76.3v199.3C23.5 365.9 0 396.2 0 432c0 44.2 35.8 80 80 80s80-35.8 80-80c0-34-21.2-63.1-51.2-74.6 3.1-5.2 7.8-9.8 14.9-13.4 16.2-8.2 40.4-10.4 66.1-12.8 42.2-3.9 90-8.4 118.2-43.4 14-17.4 21.1-39.8 21.6-67.9 31.6-10.8 54.4-40.7 54.4-75.9zM80 64c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16zm0 384c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zm224-320c8.8 0 16 7.2 16 16s-7.2 16-16 16-16-7.2-16-16 7.2-16 16-16z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                data-testid="pullRequests"
              >
                180
              </span>
            </p>
            <p>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="watch"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-eye fa-w-18"
                  data-icon="eye"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 576 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                data-testid="watchers"
              >
                2
              </span>
            </p>
            <p>
              <span
                class="sc-AxhUy hejoPV"
                data-testid="issue"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-info-circle fa-w-16"
                  data-icon="info-circle"
                  data-prefix="fas"
                  focusable="false"
                  height="14"
                  role="img"
                  viewBox="0 0 512 512"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                data-testid="issues"
              >
                74
              </span>
            </p>
          </div>
          <p>
            Fundraising App
          </p>
        </section>
      </div>
      <footer>
        <p>
          Icons powered by
           
          <a
            href="https://fontawesome.com/license"
            rel="noreferrer"
            target="_blank"
          >
            FontAwesome
          </a>
        </p>
        <p>
          Build with ‚ô• by
           
          <a
            href="https://github.com/arathjz"
            rel="noreferrer"
            target="_blank"
          >
            @arathjz
          </a>
           
        </p>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
